### DISCUSSION TITLE
Sonic file install

### DISCUSSION THREAD

**Discussion by wegarc** [community/community#132536] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Los enlaces estan rotos por favor revisar

https://sonic-net.github.io/SONiC/Supported-Devices-and-Platforms.html

Nosotros necesitamos el AS7712-32X  para router Edge-Core

**Comment by LiteBrite82** [comment, marked as answer]
Hi @wegarc , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
